.github/workflows/tv-os-build.yml

```diff
@@ -12,11 +12,11 @@ on:
   workflow_dispatch:
 jobs:
   build:
-    runs-on: macos-latest
+    runs-on: macos-12
```

Comment:
1. Rename this file to `tvos-build.yml` (without dash)
2. Apply changes from main (this version is outdated)